decidim-admin/app/controllers/decidim/admin/scopes_controller.rb

```diff
@@ -10,11 +10,7 @@ class ScopesController < Decidim::Admin::ApplicationController
 
       def index
         authorize! :index, Scope
-        @scopes = if parent_scope
-                    parent_scope.children
-                  else
-                    collection.top_level
-                  end
+        @scopes = parent_scope_children.order "name->'#{I18n.locale}' ASC"
```

Comment:
Our currently style is to use parentheses around `order`, even though `rubocop` can't detect that at the moment.